nodewatcher/modules/devices/tplink/cpe.py

```diff
@@ -90,16 +90,16 @@ class TPLinkCPE210(cgm_devices.DeviceBase):
     }
 
 
-class TPLinkCPE220(TPLinkCPE210):
+class TPLinkCPE220v1(TPLinkCPE210v1):
     """
-    TP-Link CPE210 device descriptor.
+    TP-Link CPE210 v1 device descriptor.
     """
 
-    identifier = 'tp-cpe220'
-    name = "CPE220"
+    identifier = 'tp-cpe220-v1'
+    name = "CPE220 v1"
 
 
-class TPLinkCPE510(TPLinkCPE210):
+class TPLinkCPE510(TPLinkCPE210v1):
```

Comment:
Is there no versioning for CPE 510? Should we make this v1 just in case?

Reply:
I have not found v2 of CPE510/520 mentioned anywhere, not even on TP Link support site(Usually new versions first pop up there).
I could make them v1 also to future proof them.